lib/registryFromSparse.nix

```
@@ -0,0 +1,23 @@
+{ registryFromDownloadUrl }:
+
+{ indexUrl
+, sha256
```

```suggestion
, configSha256
```

Nit: I'd prefer to name this `configSha256` to highlight that this is the hash of the _configuration file_ and not just the contents of the root URL or something!

---

ack!